jwst/tweakreg/tweakreg_step.py

```diff
@@ -111,6 +115,18 @@ class TweakRegStep(Step):
     def process(self, input):
         use_custom_catalogs = self.use_custom_catalogs
 
+        if self.separation <= _SQRT2 * self.tolerance:
+            raise ValueError(
```

Comment:
For use within the automated pipeline environment it would be better to simply issue an error-level log message and abort the rest of the step processing (and set the step completion status to skipped). We don't want things crashing and burning in the operations environment.